src/Formatter/Specialised/LegacyHttpFormatter.php

```diff
@@ -195,7 +234,7 @@ private function translateProperties(array $properties): CypherMap
                 $tbr[$key] = $this->translateProperties($castedValue);
             } elseif (is_array($value)) {
                 /** @var array<string, array|stdClass|scalar|null> $value */
-                $tbr[$key] = new CypherList($this->translateProperties($value)->values());
+                $tbr[$key] = new CypherList($this->translateProperties($value));
```

Comment:
Is this change intentional? It seems a bit unrelated to everything else and I don't know the package well enough to know why this happened

Reply:
Yes, it was intentional. Values triggers evaluation and thus mutation. If we just pass the iterable, it will lazily evaluate.